src/libstd/panicking.rs

```diff
@@ -518,7 +518,7 @@ pub fn update_count_then_panic(msg: Box<dyn Any + Send>) -> ! {
 
 /// A private no-mangle function on which to slap yer breakpoints.
 #[no_mangle]
-#[allow(private_no_mangle_fns)] // yes we get it, but we like breakpoints
+#[cfg_attr(stage0, allow(private_no_mangle_fns))]
 pub fn rust_panic(mut msg: &mut dyn BoxMeUp) -> ! {
```

Comment:
Yeah the `#[no_mangle]` here is sort of opportunistically trying to keep `rust_panic` as a symbol in the crate, but the new changes in this PR exported it a bit too aggressively, I've tweaked this although it will likely continue to not work too reliably.